### PR TITLE
[ADD] DCOS-2301 pip version check to install script

### PR DESCRIPTION
**Pull request by ytalashkocv**

### CODE REVIEW

**Comment by ytalashkocv** [comment]
@ssk2 Please review

**Comment by ssk2** [comment]
@ytalashkocv - how did you test this? I can't figure out how to install an old version of pip.

File: bin/install/install-dcos-cli.sh [hunk 1]
```diff
@@ -7,11 +7,26 @@ usage()
   echo "$(basename "$(test -L "$0" && readlink "$0" || echo "$0")") <installation-path> <dcos-url>"
 }
 
+check_pip_version()
+{
+    PIP_INFO=$(pip -V);
+    REGEX="([0-9]+\.[0-9]+)";
+    [[ $PIP_INFO =~ $REGEX ]];
+    PIP_VERSION="${BASH_REMATCH[1]}";
+    CHECK_RESULT=$(echo "$PIP_VERSION"'<='1.4 | bc -l);
```
Comment (ssk2) [inline bin/install/install-dcos-cli.sh]: This does not work on Linux - `bc` appears only be distributed with Unix.
```vagrant@mesos:~/dcos-cli$ bash install-dcos-cli.sh . http://test.com
install-dcos-cli.sh: line 16: bc: command not found
vagrant@mesos:~/dcos-cli$ bc
The program 'bc' is currently not installed. You can install it by typing:
sudo apt-get install bc```

**Comment by ssk2** [comment]
@ytalashkocv a couple of comments - this breaks the install script on Linux. Not sure how to test it either.

File: bin/install/install-dcos-cli.sh (hunk 1 above)
Comment (ytalashkocv) [inline bin/install/install-dcos-cli.sh]: Fixed.

**Comment by ytalashkocv** [comment]
@ssk2 For now version checks without `bc`. Please review.

**Comment by ssk2** [comment]
Thank you @ytalashkocv!

**Pull request merged**